x-pack/plugin/runtime-fields/qa/src/main/java/org/elasticsearch/xpack/runtimefields/test/CoreTestTranslater.java

```diff
@@ -109,24 +110,19 @@ public void execute(ClientYamlTestExecutionContext executionContext) throws IOEx
                 Map<String, String> params = Map.of("name", "hack_dynamic_mappings", "create", "true");
                 List<Map<String, Object>> dynamicTemplates = new ArrayList<>();
                 for (String type : RUNTIME_TYPES) {
-                    if (type.equals("ip")) {
-                        // There isn't a dynamic template to pick up ips. They'll just look like strings.
+                    if (type.equals(IpFieldMapper.CONTENT_TYPE)
+                        || type.equals(GeoPointFieldMapper.CONTENT_TYPE)
+                        || type.equals(KeywordFieldMapper.CONTENT_TYPE)) {
+                        // There isn't a dynamic template to pick up ips and geopoints. They'll just look like strings.
```

Comment:
It's probably worth a special comment for `keyword` here.

Reply:
I had already written a comment for keyword one line below, anyways I updated those comments.